scenes/game/local_game.gd

```diff
@@ -2941,6 +3128,8 @@ func handle_strike_attack_cleanup(performing_player : Player, card):
 	var other_player = _get_player(get_other_player(performing_player.my_id))
 	if performing_player.is_set_aside_card(card.id):
 		events += [create_event(Enums.EventType.EventType_SetCardAside, performing_player.my_id, card.id)]
+	elif performing_player.strike_stat_boosts.discard_attack_on_cleanup:
```

Comment:
Maybe the seal case should go before this discard one, just in case something wants to seal and discard? It should probably get sealed.